crates/agentgateway/src/telemetry/metrics.rs

```diff
@@ -170,6 +183,61 @@ impl Metrics {
 			gen_ai_request_duration,
 			gen_ai_time_per_output_token,
 			gen_ai_time_to_first_token,
+
+			response_bytes: {
+				let m = Family::<HTTPLabels, _>::default();
+				registry.register_with_unit(
+					"response",
+					"Total HTTP response bytes sent",
+					Unit::Bytes,
+					m.clone(),
+				);
+				m
+			},
+			tcp_downstream_rx_bytes: {
+				let m = Family::<TCPLabels, _>::default();
+				registry.register_with_unit(
+					"downstream_received",
+					"Total TCP bytes received per connection labels",
+					Unit::Bytes,
+					m.clone(),
+				);
+				m
+			},
+			tcp_downstream_tx_bytes: {
+				let m = Family::<TCPLabels, _>::default();
+				registry.register_with_unit(
+					"downstream_transmitted",
```

Comment:
```suggestion
					"downstream_sent",
```

maybe I am biased but I like to follow the Istio ones which seem to be pretty well-known. I am fine either way though